MAESTRO/scATAC_FragmentReshape.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# @Author: Dongqing Sun
+# @E-mail: Dongqingsun96@gmail.com
+# @Date:   2020-07-17 07:37:07
+# @Last Modified by:   Dongqing Sun
+# @Last Modified time: 2020-07-17 20:32:22
+
+
+import argparse
+import pysam
+import time, os
+from collections import defaultdict
+
+chr_list = ['chr'+str(i) for i in list(range(1,23))] 
```

Comment:
I do not see the chr_list variable is used afterward.
Also, for human it is chr 1:23 + X + Y
for mouse it is chr1:20 + X + Y

Reply:
I can delete the chr_list variable.